mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
Are there additional places in the code where it could be used?  I only searched for the literal values of "mm" and "in" and "sp", etc. Looks like I missed these in inspectorImage.cpp here, but that's all I see:
https://github.com/musescore/MuseScore/blob/master/mscore/inspector/inspectorImage.cpp#L180-L212